template/v1/Dockerfile

```diff
@@ -101,7 +101,8 @@ RUN \
         sagemaker-code-editor --install-extension "${ext}" --extensions-dir "${extensionloc}" --server-data-dir /opt/amazon/sagemaker/sagemaker-code-editor-server-data --user-data-dir /opt/amazon/sagemaker/sagemaker-code-editor-user-data; \
        done \
     # Copy the settings
-    && cp /etc/code-editor/code_editor_machine_settings.json /opt/amazon/sagemaker/sagemaker-code-editor-server-data/data/Machine/settings.json
+    && cp /etc/code-editor/code_editor_machine_settings.json /opt/amazon/sagemaker/sagemaker-code-editor-server-data/data/Machine/settings.json \
+    && cp /etc/code-editor/code_editor_user_settings.json /opt/amazon/sagemaker/sagemaker-code-editor-server-data/data/User/settings.json
```

Comment:
other reviewers FYI - I found that each setting has a ConfigurationScope that dictates which settings file(s) it can be defined in

All scopes here - [link](https://github.com/microsoft/vscode/blob/587ccd9b59ec6d585368be9ff1a7953600d16174/src/vs/platform/configuration/common/configurationRegistry.ts#L127-L152)

Scope for extensions.autoUpdate is set [here](https://github.com/microsoft/vscode/blob/587ccd9b59ec6d585368be9ff1a7953600d16174/src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts#L126-L140)